File: prs.5SAudits.lib/Models/Audits.cs
﻿namespace prs_5SAudits.lib.Models
{
    public record struct Audits
    {
        public int? ID { set; get; }
        public DateTime DateStarted { set; get; }
        public int Employee_ID { set; get; }
        public DateTime? DateCompleted { set; get; }
        public int Zone_ID { set; get; }
        public double OverallScore { set; get; }
        public int AuditStatus_ID { set; get; }
        public string? Notes { set; get; }
        public bool? IsDeleted { set; get; }
        public string AuditNumber { set; get; }
    }
}

File: prs.5SAudits.web/src/models/Audits.ts
export interface Audits {
    id: number | null;
    dateStarted: Date;
    employee_ID: number;
    dateCompleted: Date | null;
    zone_ID: number;
    overallScore: number;
    auditStatus_ID: number;
    notes: string | null;
    zoneName?: string;
    isDeleted: boolean;
    auditNumber: string;
}